src/skibidi_orm/query_engine/model/base.py

```diff
@@ -0,0 +1,117 @@
+""" A model class for mapping database table """
+
+from pydantic import BaseModel
+from pydantic.main import ModelMetaclass
+from typing import Any, Tuple, Dict, List
+import inspect
+import bisect
+
+def convert_name_to_table(name: str) -> str:
+    return name
+
+def check_db_name_is_correct(db_table_name: str) -> bool:
+    return True
+
+def _is_field(value: Any):
+    # Only call contribute_to_class() if it's bound.
+    return not inspect.isclass(value) and hasattr(value, "contribute_to_class")
+
+class MetaOptions:
+    """ A subsidiary to storage additional information"""
+    def __init__(self, meta: Dict[str, Any]):
+        self.fields = []
```

Comment:
I can't find where you use it. Maybe you forgot to remove it, it looks like duplicate of local_field

Reply:
Yeah, i will remove it